### DISCUSSION TITLE
Breaking change 3.1 PngMetaData HasTransparency

### DISCUSSION THREAD

**Discussion by svenclaesson** [SixLabors/ImageSharp#2606] [Q&A] [answered]
pngMetaData.HasTransparency is no longer avaliable between 3.0.2 to 3.1.
is the new way is to check pngMetaData.TransparentColor != null ?

**Comment by svenclaesson** [comment]
Maybe add a note about it[ **here** ](https://sixlabors.com/posts/announcing-imagesharp-310/#breaking-changes)?

**Comment by JimBobSquarePants** [comment, marked as answer]
Yes see https://github.com/SixLabors/ImageSharp/pull/2485
